src/main/java/com/hierynomus/sshj/userauth/keyprovider/OpenSSHKeyV1KeyFile.java

```diff
@@ -28,6 +28,7 @@
 import org.bouncycastle.asn1.nist.NISTNamedCurves;
 import org.bouncycastle.asn1.x9.X9ECParameters;
 import org.bouncycastle.jce.spec.ECNamedCurveSpec;
+import org.bouncycastle.openssl.EncryptionException;
```

Comment:
I'd rather not use an exception from BC.

Reply:
I tried to stay as close as possible to https://github.com/hierynomus/sshj/blob/b1be9258b4f22257f5c45cabc9566e6702beb8b3/src/main/java/net/schmizz/sshj/userauth/keyprovider/PKCS8KeyFile.java#L84, but am open to any other type. Should I just define a new Exception for this purpose as an inner class of OpenSSHKeyV1KeyFile?

Reply:
Let's also change this occurrence with the same exception, to keep them in sync, which is definitely a good idea ;)

Reply:
I created a new `KeyDecryptionFailedException` and throw it from both `PKCS8KeyFile` and `OpenSSHKeyV1KeyFile`.